chapters/beam_loader.asciidoc

```diff
@@ -49,3 +49,231 @@ e.g. +erts/emulator/x86_64-unknown-linux-gnu/opt/smp/+).
 
 The same program (beam_makeops) also generates the Erlang code for the
 compiler back end +beam_opcodes.erl+.
+
+=== Understanding ops.tab
+
+// Missing description about build specific options, i.e. the %unless construct
```

Comment:
Please use the WARNING admonition instead of a comment (so that we can more easily grep for these in the future). As a plus, for every TODO item a GitHub issue should exist.